odxtools/parameters/parameterbase.py

```diff
@@ -108,15 +108,16 @@ def decode_from_pdu(self, decode_state: DecodeState):
         pass
 
     def encode_into_pdu(self, encode_state: EncodeState) -> bytes:
-        """Insert the encoded value of a parameter into the coded RPC.
+        """Insert the encoded value of a parameter into a binary blob of a message.
```

Comment:
you are not inserting here, right? you are returning

Reply:
yep. I have trouble coming up with a better name, though. Do you have any ideas?

Reply:
```suggestion
        """Encode the value of a parameter into a binary blob and return it
```